arkouda/pdarraycreation.py

```diff
@@ -490,7 +490,7 @@ def ones(
     # check dtype for error
     if dtype_name not in NumericDTypes:
         raise TypeError(f"unsupported dtype {dtype}")
-    repMsg = generic_msg(cmd="create", args={"dtype": dtype_name, "size": size})
+    repMsg = generic_msg(cmd="create1D", args={"dtype": dtype_name, "shape": size})
```

Comment:
Should we make these client-side functions handle the dimensionality as well? I don't think there are instructions in the PR for how to get the client to work with ND arrays, but if we did the `"create" + str(ndim) + "D"` or something like that to build up the command string, that should just work with the increased number of dimensions I think.

Reply:
I was thinking about that, but I didn't want replace the `size` argument on `ak.zeros` and `ak.ones` with a `shape` argument for now because I wasn't sure if it's okay to modify those interfaces. Would that change be allowed?

I can definitely parameterize the calls to `set` by the arrays dimension without any interface changes though.